src/data/analytics.js

```diff
@@ -9,8 +9,11 @@ init()
 function init() {
   loadLib()
 
-  ;(window).dataLayer = (window).dataLayer || []
-  window.gtag = () => (window).dataLayer.push(arguments)
+  window.dataLayer = window.dataLayer || []
+
+  window.gtag = function (){
+    dataLayer.push(arguments)
```

Comment:
Should this be `window.dataLayer.push(arguments)`? I see that Deep Source flagged this line with "use of undeclared variables". It also flagged it with "use rest parameters instead of 'arguments'". I have no idea what that means.

Reply:
I can't find a public link to the directions but here is a screenshot.
![image](https://github.com/silinternational/wecarry-ui/assets/70765247/fa74965a-ba67-4af8-a885-b0d4f81a7c1e)
I've checked and window.dataLayer === dataLayer, but for some reason it wasn't working in one of our other apps until I made this change. I don't know about the arguments vs rest paramters, but I thought sticking close to the instructions was the best bet.